backend/src/API/mailAPI.ts

```diff
@@ -50,9 +51,65 @@ const emailAdmins = async (req: Request, subject: string, text: string) => {
   });
 };
 
+const emailMember = async (req: Request, member: IdolMember, subject: string, text: string) => {
+  const url = getSendMailURL(req);
+  const idToken = req.headers['auth-token'] as string;
+  const requestBody = {
+    subject,
+    text
+  };
+
+  return axios.post(
+    url,
+    { ...requestBody, to: member.email },
+    { headers: { 'auth-token': idToken } }
+  );
+};
+
 export const sendMemberUpdateNotifications = async (req: Request): Promise<Promise<void>[]> => {
   const subject = 'IDOL Member Profile Change';
   const text =
     'Hey! A DTI member has updated their profile on IDOL. Please visit https://idol.cornelldti.org/admin/member-review to review the changes.';
   return emailAdmins(req, subject, text);
 };
+
+export const sendTECReminder = async (req: Request, member: IdolMember): Promise<AxiosResponse> => {
```

Comment:
Since you are creating variables with promises, I'd await them first instead of inlining the `await`, otherwise you might as well just inline the value and delete the variable entirely.

So instead of
```
  const allEvents = getAllTeamEvents(req.body);
  const futureEvents = (await allEvents).filter((event) => {
```
try

```
  const allEvents = await getAllTeamEvents(req.body);
  const futureEvents = allEvents.filter((event) => {
```

Same with `memberEvents`, although I'd personally rename this to `memberEventAttendance`